src/OrbitGl/App.cpp

```diff
@@ -599,11 +599,7 @@ void OrbitApp::OnErrorEnablingUserSpaceInstrumentationEvent(
     main_window_->AppendToCaptureLog(MainWindowInterface::CaptureLogSeverity::kSevereWarning,
                                      GetCaptureTimeAt(error_event.timestamp_ns()), message);
     if (!IsLoadingCapture()) {
-      constexpr const char* kDontShowAgainErrorEnablingUserSpaceInstrumentationWarningKey =
-          "DontShowAgainErrorEnablingUserSpaceInstrumentationWarning";
-      main_window_->ShowWarningWithDontShowAgainCheckboxIfNeeded(
-          "Could not enable user space instrumentation", message,
-          kDontShowAgainErrorEnablingUserSpaceInstrumentationWarningKey);
+      SendWarningToUi("Could not enable user space instrumentation", message);
```

Comment:
One small thought that I'll simply leave to you: "User space instrumentation" is only the "internal" name now, do we want to expose it here? (This might be fine and a much bigger issue to tackle if the error messages sent from the service also contain the internal name.)

Reply:
done